src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Operations/ConvertData/ContainerRegistryTemplateProviderTests.cs

```diff
@@ -47,7 +47,7 @@ private IConvertDataTemplateProvider GetDefaultTemplateProvider()
             var convertDataConfig = new ConvertDataConfiguration
             {
                 Enabled = true,
-                OperationTimeout = TimeSpan.FromMilliseconds(50),
+                OperationTimeout = TimeSpan.FromMilliseconds(1000),
```

Comment:
>FromMilliseconds [](start = 44, length = 16)

super nit: FromSeconds(1) look cleaner for me :D

Reply:
Sure. Fixed.